api/src/env.ts

```diff
@@ -175,12 +175,31 @@ function getEnvironmentValueByType(envVariableString: string) {
 function processValues(env: Record<string, any>) {
 	env = clone(env);
 
-	for (const [key, value] of Object.entries(env)) {
+	for (let [key, value] of Object.entries(env)) {
+		// If key ends with '_FILE', try to get the value from the file defined in this variable
+		// and store it in the variable with the same name but without '_FILE' at the end
+		let newKey;
+		if (key.length > 5 && key.endsWith('_FILE')) {
+			newKey = key.slice(0, -5);
```

Comment:
Maybe this could be the point to do a `const logger = require('./logger')`?

Reply:
Unfortunately, this doesn't work either (same error as when using import on top):
`TypeError: logger.warn is not a function`

Reply:
Oh my bad, it would be `const { default: logger } = require('./logger');` 🤔

Reply:
Hmm, I don't think this is going to work:

```
const { default: logger } = require('./logger.ts');
console.log(logger);
```
Output:
```
undefined
```

Same happens with the import statement:
```
`import logger from './logger';
console.log(logger);
```
Output:
```
undefined
```

I still think this happens because it is a circular dependency (env.ts is getting imported in logger.ts).

Reply:
Workaround in `logger.ts`:
```diff
-import env from './env';
+
+// Prevent circular dependency problem with 'env.ts'
+let env;
+try {
+       env = require('./env');
+} catch {
+       env = {};
+}
```

Would you be happier with this? 😄

Reply:
Nah, lets keep it simple and go with this: https://github.com/directus/directus/pull/6101#discussion_r648493117